pkg/services/k8s/apiserver/entitystorage.go

```diff
@@ -147,7 +147,7 @@ func (s *entityStorage) write(ctx context.Context, grn *entity.GRN, uObj *unstru
 func getItems(listObj runtime.Object) ([]runtime.Object, error) {
 	out := make([]runtime.Object, 0)
 
-	switch list := listObj.(type) {
+	switch list := listObj.DeepCopyObject().(type) {
```

Comment:
is this expensive?

Reply:
yeah, we shouldn't actually do this part

Reply:
We don't have to DeepCopyObject(), it was convenient in the moment. A type assertion and `.Object` will do for this one.